buildenv/1.3.x/common/qt5/qt5.build

```diff
@@ -28,53 +29,32 @@ function prepare {
 
 	echo "# dummy" > qtbase/.gitignore
 
-	# Disable qmacpasteboardmime. There are symbol clashes with the 'cocoa' plugin,
-	# so seemingly, these two modules aren't currently engineered to be used in a
-	# static build together.
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-macextras-disable-qmacpasteboardmime.patch
-
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-force-qtimageformats-jasper-no-libjpeg.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-qtimageformats-jasper-4-color-rct-debian.patch
-
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-configureapp-use-msvc2013-mkspec.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-mariadb-support.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-mkspecs-common-msvc-desktop-mumble-debug+opt.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-disable-qtsvg-filesystem-access.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-avoid-j-underflow-in-harfbuzz.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-we-want-a-buffer-for-named-pipes.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt5-quick-and-dirty-hack-to-avoid-ssl-error-poisoning.patch
-	patch -p1 < ${MUMBLE_BUILDENV_COMMON}/qt5/patches/qt-5.6.1-work-around-client-verification-error.patch
```

Comment:
Don't drop this.

Reply:
Is it still required?

Reply:
I don't know for sure. But I don't think just before a release is a time to drop something like this.